checkpoint_quantum_firewall/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# CHANGELOG - checkpoint_quantum_firewall
+
+## 1.0.0 / 2024-01-03
+
+***Added***:
+
+* Initial Release
```

Comment:
Please add the towncrier part: https://github.com/DataDog/integrations-core/pull/16499/files#r1441760555

Reply:
Done